tekton/pipeline_run_test.go

```diff
@@ -106,14 +137,15 @@ var _ = Describe("PipelineRun", func() {
 
 		It("Can create a ReleasePipelineRun", func() {
 			Expect(releasePipelineRun.ObjectMeta.Name).
-				Should(MatchRegexp(PipelineRunPrefixName + `-[a-z0-9]{5}`))
-			Expect(releasePipelineRun.ObjectMeta.Namespace).To(Equal(Namespace))
+				Should(MatchRegexp(pipelineRunPrefixName + `-[a-z0-9]{5}`))
```

Comment:
I agree. Although I think once we stick with a format we should make sure it is consistent. For instance, here it says `[a-z0-9]{5}`. If we change this to an `uuid` or a `md5` for instance, we should make sure that it won't accept something different than that.

Reply:
TLDR: I will change it

Long version: 

The objects that were created by `EnvTest` were always in the format `GenerateName-[a-z0-9]{5}`. What I meant was:  if the suffix format (I don't know from where it comes) changes from `[a-z0-9]{5}` to something else as an `uuid` or a `md5` hash (because 5 chars in the suffix seems too easy to clash) we *eventually* would need to make sure the generated names match the pattern.

That said, it is indeed irrelevant as I the suffix is generated by k8s itself. 

I started thinking that we were managing this somehow as you mentioned "_revisit because the name generation rules change in the future_". So I guessed we were doing atm more than just using the `GenerateName`.